CIP-1694/README.md

```diff
@@ -0,0 +1,1186 @@
+* the SPOs approve of the action (the stake controlled by the SPOs who vote 'Yes' meets a certain threshold over the total registered voting stake)
+
+> **Warning**
+> As explained above, different stake distributions apply to DReps and SPOs.
+
+A successful election of a new constitutional committee, a constitutional change or a hard-fork delays
+ratification of all other governance actions until the first epoch after their enactment. This gives
+a new constitutional committee enough time to vote on current proposals, re-evaluate existing proposals
+with respect to a new constitution, and ensures that the in principle arbitrary semantic changes caused
+by enacting a hard-fork do not have unintended consequences in combination with other actions.
+
+##### Requirements
+
+The following table details the ratification requirements for each governance action scenario. The columns represent:
+
+* **Governance action type**<br/>
+  The type of governance action. Note that the protocol parameter updates are grouped into three categories.
+
+* **Constitutional committee (abbrev. CC)**<br/>
+  A value of ✓ indicates that the constitutional committee must approve this action.<br/>
+  A value of - means that constitutional committee votes do not apply.
+
+* **DReps**<br/>
+  The DRep vote threshold that must be met as a percentage of *active voting stake*.<br/>
+  A value of - means that DReps votes do not apply.
+
+* **SPOs**<br/>
+  The SPO vote threshold which must be met as a percentage of the stake held by all stake pools.<br/>
+  A value of - means that SPO votes do not apply.
+
+| Governance action type                                            | CC   | DReps    | SPOs     |
+| :---                                                              | :--- | :---     | :---     |
+| 1. Motion of no-confidence                                        | \-   | $P_1$    | $Q_1$    |
+| 2<sub>a</sub>. New committee/threshold (_normal state_)           | \-   | $P_{2a}$ | $Q_{2a}$ |
+| 2<sub>b</sub>. New committee/threshold (_state of no-confidence_) | \-   | $P_{2b}$ | $Q_{2b}$ |
+| 3. Update to the Constitution                                     | ✓    | $P_3$    | \-       |
+| 4. Hard-fork initiation                                           | ✓    | $P_4$    | $Q_4$    |
+| 5<sub>a</sub>. Protocol parameter changes, network group          | ✓    | $P_{5a}$ | \-       |
+| 5<sub>b</sub>. Protocol parameter changes, economic group         | ✓    | $P_{5b}$ | \-       |
+| 5<sub>c</sub>. Protocol parameter changes, technical group        | ✓    | $P_{5c}$ | \-       |
+| 5<sub>d</sub>. Protocol parameter changes, governance group       | ✓    | $P_{5d}$ | \-       |
+| 6. Treasury withdrawal                                            | ✓    | $P_6$    | \-       |
+| 7. Info                                                           | ✓    | $P_7$    | \-       |
```

Comment:
Why can't SPOs vote for a `Info` governance action?

Reply:
good catch, this came up before and I meant to change it so that they could. I see no reason not to let them vote too. There is also the question of whether or not the thresholds for the info action make sense.